lib/call.ts

```diff
@@ -26,6 +26,9 @@ class CallTranspiler extends base.TranspilerBase {
           this.emit('new');
         }
         var newExpr = <ts.NewExpression>node;
+        if (newExpr.arguments === undefined) {
```

Comment:
It doesn't look like it is a TS error --- TS generates the right code from it, with no warnings